DependencyInjection/Configuration.php

```diff
@@ -35,6 +35,7 @@ public function getConfigTreeBuilder()
                         ->scalarNode('recipient')->isRequired()->cannotBeEmpty()->end()
                         ->scalarNode('sender')->isRequired()->cannotBeEmpty()->end()
                         ->scalarNode('subject')->isRequired()->cannotBeEmpty()->end()
+                        ->booleanNode('send_on_warning')->defaultTrue()->end()
```

Comment:
Yeah, I was thinking about making it a "threshold flag" but when a checks return value is unknown I think that means the check was setup wrong - it isn't returning an expected value... Figured the user should be notified immediately.

What do you think?